src/py42/sdk/queries/__init__.py

```diff
@@ -5,17 +5,17 @@ class BaseQuery(object):
     def __init__(self, *args, **kwargs):
         self._filter_group_list = list(args)
         self._group_clause = kwargs.get(u"group_clause", u"AND")
-        self.page_size = 10000
+        self.page_number = kwargs.get(u"page_number", 1)
```

Comment:
This kind of feeds is relates the problem I listed here: https://github.com/code42/py42/pull/187/files#r484455200

We should either _only_ set these properties when they are not None (so not providing a default value in the `get()` and only setting if `is not None`) --- which I think is a good idea since we know the default values are already getting set somewhere.... OR we should define constants for default `PG_NUM_START` and default `SECURITYDATA_PG_SIZE`.

Do you get what I am getting at?

Reply:
I think so. Given how some other similar values are set, could we do something like this?

```
PG_NUM_START = 1
SECURITYDATA_PG_SIZE = 10000

self.page_number = page_number or PG_NUM_START
self.page_size = page_size or SECURITYDATA_PG_SIZE
```

Reply:
Perhaps we could add the variable defining page size for security events to `sdk.settings`? 

Then `FileEventQuery` and `BaseQuery` could pull it from that setting? That would allow it to exist in a single place, and also for the user to define it like they would do with `settings.items_per_page`, but specifically for security queries.

Reply:
Yeah that makes sense